pkg/driver/distro/distro.go

```diff
@@ -189,48 +192,58 @@ func Build(ctx context.Context,
 	driverType drivertype.DriverType,
 	driverVer string,
 ) (string, error) {
+	printer.Logger.Info("Trying to compile the requested driver")
 	driverFileName := toFilename(d, &kr, driverName, driverType)
-	destination := toLocalPath(driverVer, driverFileName, kr.Architecture.ToNonDeb())
-	if exist, _ := utils.FileExists(destination); exist {
-		return destination, ErrAlreadyPresent
+	destPath := toLocalPath(driverVer, driverFileName, kr.Architecture.ToNonDeb())
+	if exist, _ := utils.FileExists(destPath); exist {
+		return destPath, ErrAlreadyPresent
 	}
 
 	env, err := d.customizeBuild(ctx, printer, driverType, kr)
 	if err != nil {
 		return "", err
 	}
 
-	if env == nil {
-		env = make(map[string]string)
-	}
+	srcPath := fmt.Sprintf("/usr/src/%s-%s", driverName, driverVer)
+
+	// TODO shall we call fixupkernel here?
```

Comment:
Still unsure: we should build against non fixed up kernel, but eg: debian distro has some very specific fixupkernel logic and i must understand whether we can fix the logic to retrieve kernel urls in driverkit.

Reply:
Double checked: only Ubuntu driverkit builder actually uses KernelVersion; therefore we will probably need to fix it for ubuntu, but other distros are fine using an unfixed kr.